File: 1° Período - Introdução a Programação 1/Exercicios em C/Lista 5 - C/q3.c
//Da mesma forma que a multiplicação pode ser expressa como o resultado de várias adições, a
//potência pode ser representada por uma série de multiplicações. Faça um algoritmo que calcule a potência de
//um número informado pelo usuário utilizando o operador “*”.
#include<stdio.h>
int main(){
	int base, expoente, pot, i;
	printf("Base: ");
	scanf("%d", &base);
	printf("Expoente: ");
	scanf("%d", &expoente);
	pot=1;
	for(i=1; i<=expoente; i++){
		pot=base*pot;	
	}
	printf("Potencia: %d", pot);
} 

File: 1° Período - Introdução a Programação 1/Exercicios em Python/Lista 5 - Python/q3.py
#Da mesma forma que a multiplicação pode ser expressa como o resultado de várias adições, a
#potência pode ser representada por uma série de multiplicações. Faça um algoritmo que calcule a potência de
#um número informado pelo usuário utilizando o operador “*”.

base=int(input("BASE: "))
expoente=int(input("EXPOENTE: "))
pot=1
for i in range(1, expoente+1):
    pot=base*pot
print("Potencia: ", pot)
